docs/sources/flow/reference/cli/convert.md

```diff
@@ -0,0 +1,68 @@
+---
+title: grafana-agent convert
+weight: 100
+---
+
+# `grafana-agent convert` command
+
+The `grafana-agent convert` command converts a supported configuration format
+to Grafana Agent Flow River format.
+
+## Usage
+
+Usage: `grafana-agent convert [FLAG ...] FILE_NAME`
+
+If the `FILE_NAME` argument is not provided or if the `FILE_NAME` argument is
+equal to `-`, `grafana-agent convert` converts the contents of standard input. Otherwise,
+`grafana-agent convert` reads and converts the file from disk specified by the argument.
+
+The `--output` flag can be specified to write the contents of the converted
```

Comment:
Not sure why we list `--output` and `--bypass-warnings` both here and in the list below? I think there's no need to list them here, we could just have them in the list.